### ISSUE TITLE
Feature request: Ratings

### ISSUE THREAD

**Issue by foodaka** [WeAreNewt/NonFungibleTime#269]
Would be really good if users could rate after the service was completely to increase credibility in the system

**Comment by donosonaumczuk** [comment]
Absolutely. That would be a reputation/rating system, something we are encouraging the community to build on top!
